docs/index.ts

```diff
@@ -0,0 +1,97 @@
+const navMenuButton = document.getElementById("coffee-menu");
+const menuButton = document.getElementById("button-icon-burger");
+const navModal = document.getElementById("navModal")!;
+const sliderContainer = document.getElementById("sliderContainer")!;
+let indexValue:number = 1;
+let slideInterval:any;
```

Comment:
Is there no type that you can use for it? Avoid any, if you use any, its as if it was pure js.